app/models/mixins/archived_mixin.rb

```diff
@@ -2,11 +2,14 @@ module ArchivedMixin
   extend ActiveSupport::Concern
 
   included do
+    scope :deleted, -> { where(:deleted => true) }
+    scope :not_deleted, -> { where(:deleted => false) }
```

Comment:
Not trying to make your life harder, only if it helps (the initial refactorings should be easy to :+1: and merge).
[I also don't agree separate migration PRs are ideal.  IMHO atomic migration + *minimal* required changes are ideal.]

It may be neat to have a 3rd scope for both deleted and not deleted, replacing a few queries that now involve both ems_id, old_ems_id (and maybe old_ext_management_system).
Even if its new implementation will be trivial `all`, I'd love as many places as possible to explicitly say which of the 3 they mean.
Plus I think new implementation should be something less trivial like `unscoped(:deleted)` that works when coming from a scoped association?

I must say having the default `not_deleted` only when coming from EMS association worries me as error-prone...  Having the models themselves have a default_scope feels easier to reason about, although I don't know enough about it...
@Fryguy @kbrock what do you recommend?